src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int indexColor = new Random().nextInt(Colors.values().length);
```

Comment:
Do not create a new Instance of `Random` every time, make it a class-level variable